### PR TITLE
fix(lineage): Fix issue where large lineage fetches trigger 414 URI too long

### PR DESCRIPTION
**Pull request by gabe-lyons**
Issue here: https://github.com/linkedin/datahub/issues/2447

Ideally, we could send these ids as post data. However, restli client does not currently support this. As a temporary solution we can batch get our entity ids in size-limited batches.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-frontend/run/run-local-frontend-debug [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export JAVA_OPTS="
+	-Xms512m
+	-Xmx1024m
+	-Dhttp.port=$PORT
+	-Dconfig.file=$CONF_DIR/application.conf
+	-Djava.security.auth.login.config=$CONF_DIR/jaas.conf
+	-Dlogback.configurationFile=$CURRENT_DIR/logback.xml
+	-Dlogback.debug=true
+	-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005"
```
Comment (gabe-lyons) [inline datahub-frontend/run/run-local-frontend-debug:19]: intentionally committing this file to ease debug dev experience from now on.

File: gms/client/src/main/java/com/linkedin/BatchGetUtils.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public final class BatchGetUtils {
+    private BatchGetUtils() {
+        // not called
+    }
+
+    private static int batchSize = 25;
```
Comment (shirshanka) [inline gms/client/src/main/java/com/linkedin/BatchGetUtils.java:27]: not configurable?

**Review by shirshanka — COMMENTED** [review]
Overall looks fine!
I would like to see that batch size configurable though. Is there some way to accomplish that?

File: datahub-frontend/run/run-local-frontend-debug (hunk 1 above)
Comment (jjoyce0510) [reply datahub-frontend/run/run-local-frontend-debug:19]: thank you!

File: gms/client/src/main/java/com/linkedin/BatchGetUtils.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    public static <
+            U extends Urn,
```
Comment (jjoyce0510) [inline gms/client/src/main/java/com/linkedin/BatchGetUtils.java:30]: This is a lot of args

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**